pkg/controller/windowsmachine/metrics/metrics.go

```diff
@@ -84,6 +85,20 @@ func Add(ctx context.Context, cfg *rest.Config, namespace string) error {
 		return errors.Wrap(err, "could not create metrics Service")
 	}
 
+	// Create a monitoring client to hold the ServiceMonitor object
+	mclient, err := monclient.NewForConfig(cfg)
+	if err != nil {
+		return errors.Wrap(err, "could not create monitoring client")
+	}
+
+	// In the case of an operator restart, a previous SM object will be deleted and a
+	// new one will be created with the current spec. If a SM object does not yet exist,
+	// one will be created.
```

Comment:
Please mention [why](https://github.com/openshift/windows-machine-config-operator/pull/252#discussion_r560380125) we are deleting it. This should be called out in the commit message also.